src/main/scala/com/mozilla/spark/sql/hyperloglog/aggregates/HyperLogLogMerge.scala

```diff
@@ -18,7 +18,20 @@ import org.apache.spark.sql.Row
 import org.apache.spark.sql.expressions.{MutableAggregationBuffer, UserDefinedAggregateFunction}
 import org.apache.spark.sql.types._
 
+/**
+ * This class fixes HyperLogLogMerge when there are no
+ * rows, or when input rows are NULL.
+ */
 class HyperLogLogMerge extends UserDefinedAggregateFunction {
+
+  /**
+   * This HLL instance is has zero counts.
+   *
+   * scala> (new DenseHLL(12, new Bytes(Array.fill[Byte](1 << 12)(0)))).approximateSize.estimate
+   * res0: Long = 0
+   */
+  val emptyHll = new DenseHLL(12, new Bytes(Array.fill[Byte](1 << 12)(0)))
```

Comment:
Can we factor out `12` as a constant named to indicate that it is the default HLL bits?